src/olympia/addons/forms.py

```diff
@@ -397,40 +396,6 @@ class Meta:
                   'whiteboard')
 
 
-class AddonForm(happyforms.ModelForm):
-    name = forms.CharField(widget=TranslationTextInput,)
-    homepage = forms.CharField(widget=TranslationTextInput, required=False)
-    eula = forms.CharField(widget=TranslationTextInput,)
-    description = forms.CharField(widget=TranslationTextInput,)
-    developer_comments = forms.CharField(widget=TranslationTextInput,)
-    privacy_policy = forms.CharField(widget=TranslationTextInput,)
-    the_future = forms.CharField(widget=TranslationTextInput,)
```

Comment:
Don't worry, there are 106 more occurrences of this field in the repos, the future is not going anywhere :)